front-project/src/components/AccountMenu.jsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Avatar from "@mui/material/Avatar";
+import Menu from "@mui/material/Menu";
+import MenuItem from "@mui/material/MenuItem";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import Divider from "@mui/material/Divider";
+import IconButton from "@mui/material/IconButton";
+import Tooltip from "@mui/material/Tooltip";
+import Settings from "@mui/icons-material/Settings";
+import { getRandomColor } from "../utils/getRandomAccountColor";
+import { useSelector } from "react-redux";
+import { getUser } from "../features-store/auth/auth.selectors";
+
+export default function AccountMenu() {
+  const userName = useSelector(getUser).username;
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const avatarColor = getRandomColor(userName);
```

Comment:
getRandomColor means that function will return random different colors each call
in your case function returns the same value if name doesn't change so more accurate name is "nameToColor", "stringToColor"  or smth like that